qemu/tests/vm_poisoned_migration_prevention.py

```diff
@@ -0,0 +1,52 @@
+import re
+from shutil import rmtree
+
+from avocado.utils import process
+from virttest import error_context, qemu_monitor
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Handle migration prevention after VM memory poisoning
+    1) Boot VMs in source and destination hosts
+    2) Download the (internal) hwpoison tool
+    3) Executes the tool to poison some memory
+    4) Checks in the host dmesg the failure traces
+    5) Do migration and check the error message is correct
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+    hwpoison_file = params.get("hwpoison_file", "")
+    target_path = params.get("target_path")
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    vm.wait_for_login()
+
+    process.system(f"mkdir -p {target_path}", shell=True)
+    qemu_pid = vm.get_pid()
+
+    # This tool is only for internal use, replace the values if needed
+    error_context.base_context("Download the hwpoison tool", test.log.info)
+    process.system(f"wget {hwpoison_file} -P {target_path} ", shell=True)
+
+    error_context.context("Execute the hwpoison tool", test.log.info)
+    process.system(f"python3 {target_path}/hwpoison.py -p {qemu_pid}", shell=True)
```

Comment:
I think `sys.executable` is a better choice. However, if the script is host, can import the func from that, but I am okay for execute python file in python file using shell.

Reply:
@PaulYuuu updated !